### PR TITLE
Add logging response messages and other data to ClientLogInterceptor example

### PR DESCRIPTION
**Pull request by a-simeshin**
### Context 

Unfortunately on github and in the official documentation there are unfairly few examples on working with intercepting messages, and even less on intercepting data when responding from the server on the client side: response message, headers, status, trailers and other metadata.

I believe it would be a good option to show this work with a simple logging example.

### CODE REVIEW

File: examples/cloud-grpc-client/src/main/java/net/devh/boot/grpc/examples/cloud/client/LogGrpcInterceptor.java [hunk 1]
```diff
@@ -44,7 +49,39 @@
             final Channel next) {
 
         log.info(method.getFullMethodName());
-        return next.newCall(method, callOptions);
+        return new ForwardingClientCall.SimpleForwardingClientCall<ReqT, RespT>(next.newCall(method, callOptions)) {
+
+            @Override
+            public void sendMessage(ReqT message) {
+                log.debug("Request message: \n{}", message.toString());
```
Comment (ST-DDT) [inline examples/cloud-grpc-client/src/main/java/net/devh/boot/grpc/examples/cloud/client/LogGrpcInterceptor.java:56]: Please remove all the `.toString()` calls, as they waste performance when loggin is turned of and reduce readability.

**Review by ST-DDT — COMMENTED** [review]
This is a very nice addition to the examples.
Could you please update the original log messages to include some additional text such as `Received call to {}`? That way they match the new log statements more and follow best practices.

File: examples/cloud-grpc-client/src/main/java/net/devh/boot/grpc/examples/cloud/client/LogGrpcInterceptor.java (hunk 1 above)
Comment (a-simeshin) [reply examples/cloud-grpc-client/src/main/java/net/devh/boot/grpc/examples/cloud/client/LogGrpcInterceptor.java:56]: TO BE (for my own)
- [ ] log messages should follow pattern ``Received {action} with {action object}`` for example 
``Received call to sendMessage with message {}``
- [ ] remove all .toString()

Is that pattern correct? Sorry, I'm not a very experienced developer :(

Comment (ST-DDT) [reply examples/cloud-grpc-client/src/main/java/net/devh/boot/grpc/examples/cloud/client/LogGrpcInterceptor.java:56]: There is no real log pattern to follow as long as it done consistingly.
The most important part is that each log statement provides some kind of context for the log message.
Unfortunately, the original example only logged the called method without context. Your new log statements are good as they are, you only have upgrade the existing/old ones.

The `.toString()` is called conditionally by the logging implementation, sometimes with some additional formatting for lists/classes etc.

> Sorry, I'm not a very experienced developer :(

No problem at all. Everybody has to start somewhere. Reviewing and improving existing code is a very good way to improve your skills. (And it feels very nice to receive contributions once in a while 😉 )

Comment (a-simeshin) [reply examples/cloud-grpc-client/src/main/java/net/devh/boot/grpc/examples/cloud/client/LogGrpcInterceptor.java]: Thanks a lot for the explanation! I really appreciate it. 
Ready for a second review

**Review by ST-DDT — APPROVED** [review]
Looks good to me.

**Pull request merged**